data.js

```diff
@@ -74,29 +73,53 @@ exports.data = [
             label: "wwf",
         },
         description: {
-            content: "The WWF is active since 1961 and therefore a well experienced player within the environmental engagement game. The biggest Pro on WWF is that it is well known and People trust the organization. They do not have a precise focus on the climate crises but there is also a focus on it. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
+            content: "The WWF is active since 1961 and therefore a well experienced player within the environmental engagement game. The biggest pro of WWF is that it is well known and people trust the organization. They do not have a precise focus on the climate crises but there is also a focus on it. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
```

Comment:
1. Would shorten it and make it more neutral/descriptive: `The WWF is well known and people trust it`.
2. Can you add a source for the trust statement?
3. `They do not have a precise focus on the climate crises but there is also a focus on it.` would kill this sentence or substantiate it
4. `One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.` is this a goal or a demand? How are they working towards it?

Reply:
restructured. left 4 like it is